kouncil-frontend/apps/kouncil/src/app/send/send.component.ts

```diff
@@ -137,17 +136,24 @@ export class SendComponent {
 
   messageData$: Observable<MessageData> = combineLatest([
     this.messageDataService.messageData$,
-    this.messageDataService.messageData$.pipe(
-      switchMap(messageData => this.schemaFacade
-        .getExampleSchemaData$(this.servers.getSelectedServerId(), messageData.topicName))
-    )
+    this.schemaStateService.isSchemaConfigured$(this.servers.getSelectedServerId())
   ]).pipe(
-    map(([messageData, exampleData]) => ({
-      ...messageData,
-      key: messageData.key ?? JSON.stringify(exampleData.exampleKey),
-      value: messageData.value ? JSON.stringify(messageData.value, null, 2) :
-                                 JSON.stringify(exampleData.exampleValue, null, 2)
-    }))
+    switchMap(([messageData, isSchemaConfigured]) =>
+      iif(() => isSchemaConfigured,
+        this.schemaFacade.getExampleSchemaData$(this.servers.getSelectedServerId(), messageData.topicName).pipe(
+          map(exampleData => ({
+              ...messageData,
+              key: messageData.key ?? JSON.stringify(exampleData.exampleKey),
```

Comment:
te rzeczy nie moga sie odbywac w serwisie? Tak by tutaj otrzymać pewny obiekt, bez ifologi

Reply:
widze, ze tu combine i dwie rozne rzeczy. dobra to aprove